### PR TITLE
Updated finding tools in Hub

### PR DESCRIPTION
**Pull request by garimavs**
I have changed the content of the FAQ in the Hub with the link in the GTN.

### CODE REVIEW

**Review by annefou — COMMENTED** [review]
Do we have somewhere an FAQ to explain how to find tools?

**Comment by garimavs** [comment]
> Do we have somewhere an FAQ to explain how to find tools?

Not that I'm aware of, because when I was editing the content, I was informed that we keep the content about the tutorials only. To cover the _tool shed_ and _wrapping_ part, we have to create another FAQ.

**Comment by garimavs** [comment]
Dear team,

Kindly let me know if anything is amiss or if I can help with any improvement.

File: content/support/finding-tools/index.md [hunk 1]
```diff
@@ -1,21 +1,6 @@
...
-
-### Can't find the tool you want?
-
-* Try searching  the **[Tool Shed](http://toolshed.g2.bx.psu.edu)**.
-* New tools are added all the time that can be used in **[local](/admin/get-galaxy/)** or **[cloud](/cloudman/)** Galaxy instances.
-* If the tool you want to use is not wrapped yet, it could be.
-    * Should you have development resource, wrapped tool contributions are welcome additions to the *[Tool Shed](http://toolshed.g2.bx.psu.edu)**
-    * [Development Resources](/develop/)
-* Requests to wrap tools can be made by opening an issue and assigning the tag `tool request` at the [IUC master repository](https://github.com/galaxyproject/tools-iuc)
+[If a tool is missing](https://training.galaxyproject.org/training-material/faqs/galaxy/#if-a-tool-is-missing)
```
Comment (beatrizserrano) [inline content/support/finding-tools/index.md:6]:
```suggestion
## [How to find tools](https://training.galaxyproject.org/training-material/faqs/galaxy/#if-a-tool-is-missing)
```
same comment here with the links and the titles

**Review by beatrizserrano — APPROVED** [review]

**Comment by garimavs** [comment]
Thanks, @beatrizserrano, and @annefou, for helping me through this PR!

**Pull request merged**